src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.tsx

```diff
@@ -63,13 +69,13 @@ export const MemberDataListItemStructure: React.FC<IMemberDataListItemProps> = (
                     <div className="flex flex-col gap-y-2">
```

Comment:
- This here seems to be fixed sizing when `delegationCount` || `votingPower` is present.  This allows for the space between the `renderedHandle` and the `votingPower` as opposed to spacing growing under the `votingPower` when no delegation is present.

- Also, this might need to be discussed with design, but I think it might make sense to show votingPower based on whether it is undefined rather than when greater than zero. If not you have a very empty card hanging around.

![image](https://github.com/aragon/ods/assets/16927999/4ea8689b-1bb8-4409-ac84-c90873109ac7)

Take a look at the delegations received designs on [figma](https://www.figma.com/design/ISSDryshtEpB7SUSdNqAcw/Aragon-ODS?node-id=17598-39947&m=dev)

Reply:
Agree, updated the component 👌🏻